modules/drivers/mongo/test/metabase/driver/mongo/query_processor_test.clj

```diff
@@ -127,7 +116,8 @@
                                       {"$project" {"_id"              false
                                                    "datetime~~~month" "$_id.datetime~~~month"
                                                    "datetime~~~day"   "$_id.datetime~~~day"
-                                                   "count"            true}}],
+                                                   "count"            true}}
+                                      {"$sort" {"datetime~~~month" 1}}]
```

Comment:
The preprocessing results in an added `:order-by` clause containing the first breakout field. I'm not quite sure why not all fields are used.